lib/src/mock_query.dart

```diff
@@ -122,6 +128,37 @@ class MockQuery extends Mock implements Query {
     });
   }
 
+  @override
+  Query startAt(List<dynamic> values) =>
+      _cursorUtil(values, (docs, index) => docs.sublist(max(0, index)));
+
+  @override
+  Query endAt(List<dynamic> values) => _cursorUtil(values,
+      (docs, index) => docs.sublist(0, index == -1 ? docs.length : index));
+
+  Query _cursorUtil(
```

Comment:
Would you mind documenting this function, in particular what the f argument is supposed to be?